validators/helper.go

```diff
@@ -0,0 +1,107 @@
+package validators
+
+import (
+	"encoding/hex"
+	"fmt"
+	"strings"
+
+	"github.com/0xPolygon/polygon-edge/types"
+)
+
+func NewValidatorFromType(t ValidatorType) Validator {
+	switch t {
+	case ECDSAValidatorType:
+		return new(ECDSAValidator)
+	case BLSValidatorType:
+		return new(BLSValidator)
+	}
+
+	return nil
+}
+
+func NewValidatorSetFromType(t ValidatorType) Validators {
+	switch t {
+	case ECDSAValidatorType:
+		return new(ECDSAValidators)
+	case BLSValidatorType:
+		return new(BLSValidators)
+	}
+
+	return nil
+}
+
+func ParseValidator(t ValidatorType, s string) (Validator, error) {
+	switch t {
+	case ECDSAValidatorType:
+		return ParseECDSAValidator(s)
+	case BLSValidatorType:
+		return ParseBLSValidator(s)
+	default:
+		return nil, fmt.Errorf("invalid validator type: %s", t)
+	}
+}
+
+func AddressesToECDSAValidators(addrs ...types.Address) *ECDSAValidators {
+	set := make(ECDSAValidators, len(addrs))
+
+	for idx, addr := range addrs {
+		set[idx] = &ECDSAValidator{
+			Address: addr,
+		}
+	}
+
+	return &set
+}
+
+func ParseECDSAValidators(ss []string) (*ECDSAValidators, error) {
+	set := make(ECDSAValidators, len(ss))
+
+	for idx, s := range ss {
+		val, err := ParseECDSAValidator(s)
+		if err != nil {
+			return nil, err
+		}
+
+		set[idx] = val
+	}
+
+	return &set, nil
+}
+
+func ParseBLSValidator(s string) (*BLSValidator, error) {
```

Comment:
Please add additional documentation on this method.

What is `s`? What kind of value are you expecting here?

I shouldn't have to read the implementation to figure out what format I need to have to use this method

Reply:
Fixed at 9df67dea8b826ad21b4f9d3b5ea767f685944929